metrics/report/report_dockerfile/scaling.R

```diff
@@ -113,12 +111,6 @@ for (currentdir in resultdirs) {
 				cdata=cbind(cdata, c3)
 			}
 
-			# convert ms to seconds
```

Comment:
I added the point disabling originally when I was doing test runs of 1000 pods - at which 'point' (pun intended...) all the points merge into each other, and you end up with very thick lines the width of the points....
I'm trying to do a 'large n' run right now (100 pods), to see what this update to point size etc. looks like.